client/_library.sass

```diff
@@ -0,0 +1,15 @@
+
+// Colour Scheme	
+$main-colour: #D4E6E3 !default
+$header-colour: #9BC4BC !default
+$main-gray: #D2D2D2 !default
+
+// Spacing - TBD if we want this
+
+// Borders
+$small-border: 1px solid gray
+$main-radius: 3px
+
+@mixin rounded-border
+	border-radius: $main-radius
+	border: $small-border
```

Comment:
```suggestion
	border: $small-border
	
```

Reply:
Is there a change here? I'm not seeing it. Did I add spaces by mistake?